libs/shared/src/components/CarrierBox.vue

```diff
@@ -18,13 +18,14 @@
 
 <script lang="ts" setup>
 import {toRefs} from 'vue';
-import {type CarrierName} from '@myparcel/constants';
+import {resolveCarrierName} from '../utils';
+import {type CarrierIdentifier} from '../types';
 import {useCarrierRequest} from '../composables';
 import CarrierLogo from './CarrierLogo.vue';
 import Box from './Box.vue';
 
-const props = defineProps<{carrier: CarrierName}>();
+const props = defineProps<{carrier: CarrierIdentifier}>();
 const propRefs = toRefs(props);
 
-const {data} = useCarrierRequest(propRefs.carrier);
+const {data} = useCarrierRequest(resolveCarrierName(propRefs.carrier.value));
```

Comment:
waarom doe je nou weer `.value`? nu is het weer niet reactive

Reply:
Ik heb de naam gewijzig. Het zou nu reactive moeten zijn.